### DISCUSSION TITLE
How to load Parquet driver?

### DISCUSSION THREAD

**Discussion by donnyv** [MaxRev-Dev/gdal.netcore#138] [Q&A] [answered]
How would I go about adding the Parquet driver? 
https://gdal.org/drivers/vector/parquet.html

**Comment by MaxRev-Dev** [comment]
It's available via VCPKG https://vcpkg.io/en/package/arrow.html
You will need to update the list of drivers [here](https://github.com/MaxRev-Dev/gdal.netcore/blob/4428f2cec8c363648b718bac3185d80775941816/shared/GdalCore.opt#L30) with `"arrow[parquet"`
Then compile for the specific runtime. See Readme files in runtime directory and CI workflows.
I can try to run it in CI next week, but I can't promise publishing if it won't compile and will require more work.

**Comment by BouuuuuuH** [comment]
Hello,

Any news about parquet integration ? it would be helpful.

**Comment by MaxRev-Dev** [comment, marked as answer]
Arrow and Parquet are available now at [v3.11.5.395](https://github.com/MaxRev-Dev/gdal.netcore/releases/tag/v3.11.5.395)
